go.mod

```diff
@@ -7,9 +7,9 @@ require (
 	github.com/google/go-jsonnet v0.14.0
 	github.com/jsonnet-bundler/jsonnet-bundler v0.1.1-0.20190930114713-10e24cb86976
 	github.com/pkg/errors v0.8.1
-	github.com/prometheus/client_golang v1.1.0
-	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
-	github.com/prometheus/common v0.6.0
+	github.com/prometheus/client_golang v1.5.1
+	github.com/prometheus/client_model v0.2.0
+	github.com/prometheus/common v0.9.1
 	github.com/prometheus/prometheus v2.5.0+incompatible
```

Comment:
This dependency needs to be updated as well

Reply:
Yes, that's correct. I wanted to but was getting error and couldn't figure out how to update it because on the pkg.go.dev (https://pkg.go.dev/mod/github.com/prometheus/prometheus?tab=versions)  site the latest version is 2.5.0 and on the github page of prometheus (https://github.com/prometheus/prometheus/releases/tag/v2.17.1) the latest release is 2.17.1
Any suggestion would be helpful.

Reply:
You can try `go get github.com/prometheus/prometheus@<commit SHA of prometheus' latest stable release>`

Reply:
thanks, updated it using `go get`